frontend/public/components/filter-toolbar.tsx

```diff
@@ -258,6 +261,20 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
 
   const switchFilter = (type: FilterType) => {
     setFilterType(FilterType[type]);
+    if (FilterType[type] === 'Name') {
+      setPlaceholder(
+        nameFilterPlaceholder ||
+          (FilterType[type] === 'Name' ? 'Search by name...' : 'app=frontend'),
+      );
+    }
+
+    if (FilterType[type] === 'Label') {
+      setPlaceholder(
+        labelFilterPlaceholder ||
+          (FilterType[type] === 'Label' ? 'Search by label...' : 'app=frontend'),
```

Comment:
```suggestion
        labelFilterPlaceholder || 'app=frontend',
```

Reply:
Instead of 'app=frontend' i could keep it as 'Search by label' so that if `labelFilterPlaceholder` is not set then switching the filter keeps the placeholder appropriate. In any case QA is looking for this as stated in the bug

Reply:
I have added `app=frontend` as default